src/plugins/embeddable/public/embeddable_panel/embeddable_panel_error.tsx

```diff
@@ -51,6 +53,22 @@ export function EmbeddablePanelError({
     [label, title]
   );
 
+  const overrideDisplay = core
+    ? getSearchErrorOverrideDisplay({
```

Comment:
nit: I don't think this ternary is necessary. Core should always be defined here because this cannot happen until after the start services promise is resolved in`src/plugins/embeddable/public/embeddable_panel/use_embeddable_panel.ts`

Reply:
resolved with https://github.com/elastic/kibana/pull/170790/commits/24f5cde2cbaaeff904710f9e60b36d25c4f80c9c